### PR TITLE
Update the note of unit-stride fault-only-first loads.

### PR DESCRIPTION
**Pull request by Hsiangkai**
Address issue #81.

### CODE REVIEW

**Review by zakk0610 — APPROVED** [review]
LGTM

File: rvv-intrinsic-api.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ void vwrite_csr(enum RVV_CSR csr, unsigned long value);
 #### [Intrinsic functions list](intrinsic_funcs/02_vector_loads_and_stores_functions.md#unit-stride-fault-only-first-loads-functions)
 
 #### Notes
-- Execute a regular load and stop load operation if there is 0 value of element, and then set vl equals the index of last non-zero value.
+-  The instruction traps if the first element to be loaded is at an invalid address, for example it is a null pointer or it points to a page you don't have read access to. If the address of the first element is valid, but some other element being loaded is not valid, VL will be changed to the number of elements that were able to be loaded without a trap. It is up to the user to look for 0s or another terminator value(s) in the data returned up to the new vl value.
```
Comment (topperc) [inline rvv-intrinsic-api.md:99]: I'm not sure if that's a complete list of reasons a load can trap.

Comment (Hsiangkai) [reply rvv-intrinsic-api.md:99]: Should we list all of the reasons of exceptions? Or could we just describe the behavior under the exceptions?

Comment (topperc) [reply rvv-intrinsic-api.md:99]: We can probably just use this from the V extension

```
The unit-stride fault-only-first load instruction is used to vectorize loops with data-dependent exit conditions (while loops). These instructions execute as a regular load except that they will only take a trap on element 0. If an element > 0 raises an exception, that element and all following elements in the destination vector regis- ter are not modied, and the vector length vl is reduced to the number of elements processed without a trap.
```

**Review by topperc — APPROVED** [review]
LGTM

**Pull request merged**